src/vs/workbench/contrib/terminal/test/browser/links/terminalValidatedLocalLinkProvider.test.ts

```diff
@@ -62,7 +62,11 @@ const supportedLinkFormats: LinkFormatInfo[] = [
 	{ urlFormat: '{0}({1}, {2})', line: '5', column: '3' },
 	{ urlFormat: '{0} ({1}, {2})', line: '5', column: '3' },
 	{ urlFormat: '{0}:{1}', line: '5' },
+	{ urlFormat: '{0}:{1}:foo', line: '5' },
+	{ urlFormat: '{0}:{1} foo', line: '5' },
 	{ urlFormat: '{0}:{1}:{2}', line: '5', column: '3' },
+	{ urlFormat: '{0}:{1}:{2}:foo', line: '5', column: '3' },
+	{ urlFormat: '{0}:{1}:{2} foo', line: '5', column: '3' },
```

Comment:
I think this might actually be testing that the :foo gets included in the link 🤔 

We might actually need a special case test like with the git diff links for this problem:

https://github.com/microsoft/vscode/blob/d5a670eaeb0f5f14ec29d42276c8f9bc97102884/src/vs/workbench/contrib/terminal/test/browser/links/terminalValidatedLocalLinkProvider.test.ts#L122-L129